apps/condo/domains/common/components/containers/BaseLayout/AuthLayout.tsx

```diff
@@ -0,0 +1,246 @@
+import { PageHeader as AntPageHeader, PageHeaderProps } from 'antd'
+import React from 'react'
+import Router from 'next/router'
+import { FormattedMessage } from 'react-intl'
+import { Layout } from 'antd'
+const { Footer, Content } = Layout
+import { Logo } from '@condo/domains/common/components/Logo'
+import { SUPPORT_EMAIL, SUPPORT_PHONE } from '@condo/domains/common/constants/requisites'
+
+const LINK_STYLE = { color: '#389E0D' }
+
+const PageContent: React.FC = ({ children }) => {
+    return (
+        <Content style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: 'calc(100vh - 200px)', minWidth: '500px' }}>
+            {children}
+        </Content>
+    )
+}
+
+const PageFooter: React.FC = () => {
+    return (
+        <Footer style={{ color: 'gray', backgroundColor: 'white', fontSize: '12px', lineHeight: '20px' }}>
```

Comment:
from constns

Reply:
Moved to constants, used color from colors constant.